pepper-apis/src/main/java/org/broadinstitute/ddp/cache/IdToCacheKeyMapper.java

```diff
@@ -0,0 +1,7 @@
+package org.broadinstitute.ddp.cache;
+
+import org.jdbi.v3.core.Handle;
+
+public interface IdToCacheKeyMapper<K> {
+    K mapToKey(Long id, Handle handle);
+}
```

Comment:
These are the interfaces we want cache users to provide if they want auto-eviction of invalidated keys. Examples of usage below.